test/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+installer:
+  image: gruntwork/gruntwork-installer
+  volumes:
+    # Mount the scripts in the root directory under /src
+    - ../:/src
+    # Mount the test code in this directory under /test
+    - .:/test
```

Comment:
I suppose `/src` will contain the test code too. A bit messy, but not a big concern.